css/main.css

```diff
@@ -217,9 +217,96 @@ body {
   text-align: justify;
 }
 
+.cards {
+  display: grid;
+  grid-template-columns: repeat(2, 2fr);
+  grid-auto-rows: auto;
+  grid-gap: 5rem;
+}
+.card,
+.modal-card {
+  border-radius: 20px;
+  padding: 2rem;
+  text-align: center;
+}
+.card {
+  border: 2px solid #e7e7e7;
+}
+.card:hover {
+  transform: scale(1.1);
+  cursor: pointer;
+}
+.read-more {
+  border-radius: 10px;
+  outline: none;
+  background: #e3e3e3;
+  color: #000;
```

Comment:
Can we please use CSS variables all over the code of this PR instead of hex code ??

Reply:
made the changes.